addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js

```diff
@@ -160,13 +160,25 @@ export class SpreadsheetDashboardAction extends Component {
         return url;
     }
 
+    async toggleFavorite() {
+        if (!this.state.activeDashboard) {
+            return;
+        }
+
+        const { id, isFavorite } = this.state.activeDashboard;
+        await this.orm.call("spreadsheet.dashboard", "action_toggle_favorite", [id]);
+        this.state.activeDashboard.isFavorite = !isFavorite;
+    }
+
     toggleSidebar() {
         this.state.sidebarExpanded = !this.state.sidebarExpanded;
     }
 
     get activeDashboardGroupName() {
-        return this.getDashboardGroups().find((group) =>
-            group.dashboards.some((d) => d.id === this.activeDashboardId)
+        return this.getDashboardGroups().find(
+            (group) =>
+                group.name !== "FAVORITES" && // Skip the FAVORITES group
```

Comment:
```suggestion
                group.id !== "favorites" && // Skip the FAVORITES group
```
the name won't work because of translations :wink: